modules/appeals_api/config/schemas/v2/200996_headers.json

```diff
@@ -49,7 +49,10 @@
     "X-VA-Last-Name": {
       "allOf": [
```

Comment:
I think that since you're not doing any references that you could get away with removing `allOf` and just combining all the keys into `X-VA-Last-Name: {...}`

Reply:
Refactoring breaks it :(

Reply:
Create a separate ticket to look into the structure?